.github/workflows/ci_insights.yml

```diff
@@ -58,13 +52,7 @@ jobs:
         run: sed -i.bak 's/ENABLE_SIGNING=1/ENABLE_SIGNING=0/' .compose.env
 
       - name: build stack
-        run: make docker/build
-
-      - name: run migrations
-        run: make docker/migrate
-
-      - name: load test data
-        run: make docker/loaddata
+        run: make docker/all
```

Comment:
I kind of like the idea of these steps being separate in the process (and more so in the action runs output). 

Is there an advantage to smashing it into one?  
Is it to avoid duplicating 'make docker/all' logic in multiple places?

Reply:
Because new developers already have too many things to learn without also having to remember what order to run the make targets that get the stack ready.